packages/apps/shopify-api/lib/billing/check.ts

```diff
@@ -133,6 +133,7 @@ const HAS_PAYMENTS_QUERY = `
         id
         name
         test
+        status
```

Comment:
We were not returning the status from this method. So we had no way to know which subscription was actually active.